charts/pega/config/deploy/prlog4j2.xml

```diff
@@ -159,6 +171,9 @@
     <Logger name="com.pega.dsm.dnode.impl.dataflow.service.DataFlowDiagnosticsFileLogger" additivity="true" level="info">
         <AppenderRef ref="DATAFLOW"/>
     </Logger>
+    <Logger name="com.pega.platform.service.dds.client.metrics.reporters.DdsMetricsLogger" additivity="false" level="info">
```

Comment:
This should be `additivity="true"` to ensure that the console appender also receives the log messages.